app/middlewares/rack/domain_redirect.rb

```diff
@@ -1,42 +1,60 @@
-      ['cs.', '', /cs.cwc.im$/], # TEMP: work out better general purpose locale subdomain + short domain redirect
-      ['cz.', '', /cz.cwc.im$/], # TEMP: work out better general purpose locale subdomain + short domain redirect
-      ['da.', '', /da.cwc.im$/], # TEMP: work out better general purpose locale subdomain + short domain redirect
-      ['de.', '', /de.cwc.im$/], # TEMP: work out better general purpose locale subdomain + short domain redirect
-      ['en.', '', /en.cwc.im$/], # TEMP: work out better general purpose locale subdomain + short domain redirect
-      ['es.', '', /es.cwc.im$/], # TEMP: work out better general purpose locale subdomain + short domain redirect
-      ['fi.', '', /fi.cwc.im$/], # TEMP: work out better general purpose locale subdomain + short domain redirect
-      ['fr.', '', /fr.cwc.im$/], # TEMP: work out better general purpose locale subdomain + short domain redirect
-      ['gr.', '', /gr.cwc.im$/], # TEMP: work out better general purpose locale subdomain + short domain redirect
-      ['he.', '', /he.cwc.im$/], # TEMP: work out better general purpose locale subdomain + short domain redirect
-      ['id.', '', /id.cwc.im$/], # TEMP: work out better general purpose locale subdomain + short domain redirect
-      ['in.', '', /in.cwc.im$/], # TEMP: work out better general purpose locale subdomain + short domain redirect
-      ['it.', '', /it.cwc.im$/], # TEMP: work out better general purpose locale subdomain + short domain redirect
-      ['pl.', '', /pl.cwc.im$/], # TEMP: work out better general purpose locale subdomain + short domain redirect
-      ['pt.', '', /pt.cwc.im$/], # TEMP: work out better general purpose locale subdomain + short domain redirect
-      ['ru.', '', /ru.cwc.im$/], # TEMP: work out better general purpose locale subdomain + short domain redirect
-      ['sv.', '', /sv.cwc.im$/], # TEMP: work out better general purpose locale subdomain + short domain redirect
-      ['tr.', '', /tr.cwc.im$/], # TEMP: work out better general purpose locale subdomain + short domain redirect
-      ['',    '', /cwc.im/]
+    MAIN_REDIRECT_CONFIGS = [
+      # SSfWD
+      ['', '/steal-something-from-work-day', /stealfromwork\.crimethinc\.com$/],
+      ['', '/steal-something-from-work-day', /stealfromworkday\.com$/],
+      ['', '/steal-something-from-work-day', /stealsomethingfromworkday\.com$/],
+
+      # TCE
+      ['', '/tce', /tochangeeverything\.com/],
+
+      # Heroku subdomain
+      ['', '', /crimethinc\.herokuapp\.com$/],
+
+      # Other TLDs
+      ['', '', /crimethinc\.gay$/],
+
+      # ccTLDs => localized subdomain
+      ['es.', '', /crimethinc\.es/],
+      ['de.', '', /crimethinc\.de/],
+      ['cs.', '', /cz\.crimethinc\.com/], # Fix our orignal mistaken assumption
+
+      # short domain (for historical twitter/etc posts)
+      ['', '', /cwc\.im/]
+    ].freeze
+
+    CC_TLD_SUBDOMAINS = %w[
```

Comment:
It'd be better if we could read this a file/value in the repo
And used for both this _and_ for allowed locales